### DISCUSSION TITLE
Livewire is not working without PHP artisan serve

### DISCUSSION THREAD

**Discussion by rifqiadityariadhi** [livewire/livewire#4514] [1. Help]
How to use laravel livewire without php artisan serve? I am using vhost on apache2 and using Laravel 8

**Comment by joshhanley** [comment]
@rifqiadityariadhi if you can get Laravel running on your server using Apache, then Livewire should run as it is an extension of Laravel.

So get Laravel running and you should be good to go!

Hope this helps!
